### DISCUSSION TITLE
outliner for edit mode and mindmap for preview mode?

### DISCUSSION THREAD

**Discussion by shaggyfeng** [vslinko/obsidian-outliner#66] [Ideas]
It seems the bullet-line which shows the different level of items and sub-items, only shows up in edit mode in my case. And in my using experience, outliner is very important in edit mode, but not as important in preview mode. Then I started to wonder maybe it would be very helpful to have a mindmap graph showcasing the same info in a shorter format(not showing the full sentence but the first part of the sentence) in preview mode. Not to mention the actual mindmap add-on is not as well implemented as this outliner add-on IMO. Anyway, obsidian-outliner add-on is the most wonderful add-on I have. Thank you for all the crazy hard works.

**Comment by vslinko** [comment]
@shaggyfeng thank you for your idea!

I don’t have any vision how to enhance preview mode yet. But I think it’s better to create another plugin for such functionality like mindmap in preview mode.

May be it’s better to post your idea into obsidian forum https://forum.obsidian.md/c/plugins/10
